### ISSUE TITLE
Fix password reset emails not being sent

### ISSUE THREAD

**Issue by hillaryj** [18F/cg-product#1205]
# What we're after
Customers are reporting either:
- not receiving any password emails
- not receiving password emails and then getting a bunch of them at once

## Hypothesized benefit(s)/why:
- Users will be able to reset their passwords

### Security considerations
No known security considerations as this is fixing the system to correspond to expected behavior

### Notes for implementers

- This may be related to a SMTP credential rotation that didn't propagate #1205

**Comment by hillaryj** [comment]
There are two issues here - Chris and Van and I worked on this this morning and believe it to be resolved. 

1. New users couldn't set up and log in. We found that there were UAA errors after submitting the invite for the affected users, but these may be related to browser cookies. I have a report of one of the two users we reset has already been able to log on successfully.

2. OMB had some users who couldn't get Forgot Password emails and then they got a bunch of them at once - It’s likely that the OMB issues with “forgot password” got caught up in the credential rotation on Tues/Wed so their links were invalidated before they even received them.

I'll close this issue once I get verification from the second new user that they were able to log in. We can reopen it if additional Forgot Password email issues are reported.
